enginetest/queries/column_alias_queries.go

```diff
@@ -264,9 +261,15 @@ var ColumnAliasQueries = []ScriptTest{
 			{
 				// GMS returns "expression 'dt.two' doesn't appear in the group by expressions", but MySQL will execute
 				// this query.
+				Skip:  true,
+				Query: "select 1 as a, one + 1 as mod1, dt.* from mytable as t1, (select 1, 2 from mytable) as dt (one, two) where dt.one > 0 group by one;",
+				// column names:  a, mod1, one, two
+				Expected: []sql.Row{{1, 2, 1, 2}},
+			},
+			{
 				Skip:     true,
```

Comment:
(minor) If there are any note, hints, or related GitHub issues about why this query is failing and needs to be skipped, that would be good to add in a comment here.